docker-compose.yml

```diff
@@ -45,9 +45,9 @@ services:
 #     NO_MEMORY_CACHE: 1
 #     NO_BACKUP: 1
 #     NO_REQUIRE_ACCOUNT: 1
-### Uncommenting The following lines will start RDM in lldb and store crash reports in the backups folder
-### Running in lldb will use a bit more resources. (Running RDM works like usual)
-#   entrypoint: ["/bin/sh", "-c", "lldb -b -o \"file ./RealDeviceMapApp\" -o \"breakpoint set --file main.swift --line 9\" -o \"run\" -o \"process handle SIGPIPE -n true -p true -s false\" -o \"continue\" -k \"bugreport unwind --outfile ./backups/crash-$$(date +%s).log\" -k \"exit\""]
+### Uncommenting The following lines will start RDM in lldb and store crash reports in the "backups" folder
```

Comment:
with that new solution for lldb there is no crash report stored in "backups" folder anymore :)